packages/hooks/src/useDelayedTotalAccountBalance.ts

```diff
@@ -0,0 +1,31 @@
+import useRealTotalAssetCurrency from './useTotalAssetCurrency';
+import useExchangeRate from './useExchangeRate';
+import useThrottle from './useThrottle';
+
+type TUseDelayedTotalAccountBalance = {
+    balance?: number;
+    currency?: string;
+    account_type?: string;
+};
+
+const useDelayedTotalAccountBalance = (accounts: TUseDelayedTotalAccountBalance[], delay: number) => {
+    const total_assets_real_currency = useRealTotalAssetCurrency();
+    const { getRate } = useExchangeRate();
+
+    const balance = accounts.reduce((total, account) => {
```

Comment:
this is array of `TUseDelayedTotalAccountBalance` object, right?
how first object is total second one is account?